MetaMask Plugin/How to create a MetaMask Plugin

```diff
@@ -0,0 +1,26 @@
+Introduction
+
+1. Metamask plugin what’s new? 
```

Comment:
```suggestion
1. What's new with the Metamask plugin
```

But this is specifically about the APIs yes? We don't need to cover everything that's new with Metamask unless it relates to the plugin APIs.